application/dash_application/views/SARS-CoV-2.py

```diff
@@ -399,7 +407,7 @@ def layout(query_string):
                                 df,
                                 RAW_DATA_COLUMNS,
                                 [
-                                    (avg_coverage_cutoff_label, BEDTOOLS_CALC_COL.MeanCoverage, avg_coverage_cutoff_value,
+                                    (avg_coverage_cutoff_label, BEDTOOLS_CALC_COL.MeanCoverage, initial["avg_coverage_cutoff"],
```

Comment:
The failure table is missing the failed on target.

Reply:
Fixed